lib/node_modules/@stdlib/math/generics/random/sample/lib/factory.js

```diff
@@ -88,12 +93,28 @@ var slice = Array.prototype.slice;
 *
 * out = sample({
 *     'replace': false
-*     'size': 1
 * });
-* // returns [ 0 ] or [ 1 ]
+* // returns [ 0, 1 ] or [ 1, 0 ]
```

Comment:
This will be hard to test in an automated fashion, as is. Let's go with

> // e.g., returns [ 0, 1 ] or [ 1, 0 ]

The idea behind `e.g.,` is that it signals that a `returns` annotation should be skipped, as the output cannot be known *a priori*.